data/issuerDid.ts

```diff
@@ -67,6 +67,11 @@ export const issuerDidRegistry: RegistryRaw<IssuerDidEntry> = {
       name:'College Unbound',
       location:'Providence, RI, USA',
       url:'https://collegeunbound.edu',
+    },
+    'did:key:z6MkgPBJwKZD8FeFYpps9opan1VdVAeo8EvzJc15BofGujkV':{
```

Comment:
Why the addition of this DID to the hardcoded registry? I expect this file to be deleted, no?